### DISCUSSION TITLE
Next.js에서 절대경로 alias 설정

### DISCUSSION THREAD

**Discussion by baechanghyeon** [WantedPreonboardingFE5team/pre-onboarding-7th-2-1-5#45] [General]
이번 Next.js + Typescript에서 이전에 시도 했던 절대경로 alias 설정을 Next.js 자체에서 지원을 해주네요...🥳
<img width="408" alt="스크린샷 2022-11-12 오전 1 53 05" src="https://user-images.githubusercontent.com/28972561/201390171-e0b0f0c4-db52-4955-9673-8505c9acebe0.png">
Next.js 공식문서에 왠만한건 다 있는거 같습니다!
참고 URL : [Next.js 킹식문서](https://nextjs.org/docs/advanced-features/module-path-aliases)

**Comment by jaeyeong815** [comment]
next 강의 들으면서 영업 당하고있었는데 이거까지 지원..?? 저 next로 갈아탑니다..

**Reply by baechanghyeon** [reply]
아 맞다 그리고 13버전이 아직 반영된건 아닌거 같습니다 보니까 디데이 17일 이렇게 되어있네요!
그리고 새롭게 반영되는 문법도 아직 예시가 다 올라오진 않았더라구요

**Reply by jaeyeong815** [reply]
네 맞아욥 그때까지 12버전으로 좀 익히다가 요긴하게 써먹어보겠습니다 후후후.. 좋은 정보 감사합니다..

**Reply by jaeyeong815** [reply]
ㅋㅋㅋㅋ 킹식문서 ㅋㅋㅋㅋㅋㅋㅋㅋ 아 저걸 지금봤네요 ㅎ 킹식문서 킹스트js

**Reply by baechanghyeon** [reply]
공식문서가 영어라서 많이 불편하지만 킹식문서 답게 모든 정보가 다 있는거 같네여...
